analytics/CAR-2021-01-008.yaml

```diff
@@ -0,0 +1,31 @@
+---
+title: Detect disabling of UAC via reg.exe
+submission_date: 2020/12/11
+information_domain: 'Host'
+platforms:
+  - Windows
+subtypes:
+  - Process
+analytic_types:
+  - TTP
+contributors:
+  - Cyware Labs
+id: CAR-2021-01-008
+description: |-
+  Threat actors often, after compromising a machine, try to disable User Access Control (UAC) to escalate privileges. This is often done by changing the registry key for system policies using “reg.exe”, a legitimate tool provided by Microsoft for modifying the registry via command prompt or scripts. This action interferes with UAC and may enable a threat actor to escalate privileges on the compromised system, thereby allowing further exploitation of the system.
+  - technique: T1548
+    coverage: Medium
+    subtechniques:
+      - 002
+    tactics:
+      - TA0004
+implementations:
+- name: Detect disabling of UAC via reg.exe
+
+  description:  This query looks for the specific use of reg.exe in correlation to commands aimed at disabling UAC
+  code: |-
+    index = * ParentImage = *cmd* CommandLine ="*reg.exe*" AND CommandLine = "*REG_DWORD*" AND CommandLine = "*/d 0*" "HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Policies\\System" | table _time host User CommandLine
```

Comment:
Try and use specific filepaths for executables to improve search efficiency; also include the index and event type.